libs/build_nceplibs.sh

```diff
@@ -313,6 +312,8 @@ if [[ "$name" == "crtm" ]]; then
        cp ./crtm_fix-${install_as}/SpcCoeff/Big_Endian/* $prefix/fix
        cp ./crtm_fix-${install_as}/SpcCoeff/netcdf/* $prefix/fix
        cp ./crtm_fix-${install_as}/TauCoeff/ODPS/Big_Endian/* $prefix/fix
+       mv $prefix/fix/amsua_metop-c.SpcCoeff.bin $prefix/fix/amsua_metop-c.SpcCoeff.noACC.bin
+       cp ./crtm_fix-${install_as}/SpcCoeff/Little_Endian/amsua_metop-c_v2.SpcCoeff.bin $prefix/fix/amsua_metop-c.SpcCoeff.bin
```

Comment:
Is using Little_Endian correct here? Just because all the others are using Big_Endian.